common/components/WalletDecrypt/Keystore.tsx

```diff
@@ -57,6 +60,7 @@ export default class KeystoreDecrypt extends Component {
                 type="password"
               />
             </div>
+            {isWalletLoading ? <Spinner /> : ''}
```

Comment:
Minor, but you can simplify these boolean checks to `{isWalletLoading && <Spinner />}`